src/sections/downloads-v1/docker.marko

```diff
@@ -1,57 +1,85 @@
+    </p>
+  </install-step>
+  <install-step num="1" title="Avoid common pitfalls">
+    <h5>Volumes and Paths</h5>
+    <p>
+      There are two common problems with Docker volumes: Paths that differ between the Lidarr and download client container and paths that prevent fast moves and hard links.<br/>
+      The first is a problem because the download client will report a download's path as
+      <code>/torrents/Some.Artist/</code>, but in the Lidarr container that might be at
+      <code>/downloads/Some.Artist/</code>. The second is a performance issue and causes problems for seeding torrents. Both problems can be solved with well planned, consistent paths.
+    </p>
+    <p>
+      Most Docker images suggest paths like
+      <code>/music</code>
+      and
+      <code>/downloads</code>. This causes slow moves and doesn't allow hard links because they
+      are considered two different file systems
+      <i>inside</i>
+      the container. Some also recommend paths for the download client container that are different from the Lidarr container, like
+      <code>/torrents</code>.<br/>
+      The best solution is to use a single, common volume
+      <i>inside</i>
+      the containers, such as
+      <code>/data</code>.
+      Your Music would be in
+      <code>/data/Music</code>, torrents in
+      <code>/data/downloads/torrents</code>
+      and/or usenet downloads in
+      <code>/data/downloads/usenet</code>.
+    </p>
+    <p>If this advice is not followed, you may have to configure a Remote Path Mapping in the Lidarr web UI (Settings &rsaquo; Download Clients).</p>
+    <h5>Ownership and Permissions</h5>
+    <p>
+      Permissions and ownership of files is one of the most common problems for Lidarr users, both inside and outside Docker. Most images have environment variables that can be used to override
+      the default user, group and umask, you should decide this before setting up all of your containers.
+      The recommendation is to use a common group for all related containers so that each container can use the shared group permissions to read and write files on the mounted volumes.<br/>
+      Keep in mind that Lidarr will need read and write to the download folders as well as the final folders.
+    </p>
+    <p>
+      <i>
+  For a more detailed explanation of these issues, see
+        <a href="https://wiki.servarr.com/Docker_Guide">The Best Docker Setup and Docker Guide</a>
+        wiki article.
+      </i>
+    </p>
+  </install-step>
+  <install-step num="2" title="Install Lidarr">
+    <p>
+      To install and use these Docker images, you'll need to keep the above in mind while following their documentation. There are many ways to manage Docker images and containers too, so installation
+      and maintenance of them will depend on the route you choose.
+      <ul>
+        <li>
+          <h5>
+            <a href="https://hub.docker.com/r/hotio/Lidarr">hotio/Lidarr:release</a>
```

Comment:
Same issue here with link…

Let’s change to ghcr tag as well to be consistent with our lsio tag…. Because dockerhub sucks.

also let’s link to https://hotio.dev/containers/lidarr/ or to the ghcr page.